src/App.js

```diff
@@ -1,21 +1,33 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
```

Comment:
I think since Babel does JSX now this just gets transformed as well. I could be wrong about that though.